docs/installation.rst

```diff
@@ -157,45 +257,203 @@ To build library and tests, run:
+|                                   |complex_scale_contraction_test_m5n5k5                                        |
+|                                   +-----------------------------------------------------------------------------+
+|                                   |complex_scale_contraction_test_m6n6k6                                        |
+|                                   +-----------------------------------------------------------------------------+
+|                                   |rank2_permutation_test                                                       |
+|                                   +-----------------------------------------------------------------------------+
+|                                   |rank3_permutation_test                                                       |
+|                                   +-----------------------------------------------------------------------------+
+|                                   |rank4_permutation_test                                                       |
+|                                   +-----------------------------------------------------------------------------+
+|                                   |rank5_permutation_test                                                       |
+|                                   +-----------------------------------------------------------------------------+
+|                                   |rank6_permutation_test                                                       |
++-----------------------------------+-----------------------------------------------------------------------------+
+
+Build performance
+^^^^^^^^^^^^^^^^^
+
+Depending on the resources available to the build machine and the build configuration selected, hipTensor build times can be on the order of an hour or more. Here are some things you can do to reduce build times:
+
+* Target a specific GPU (e.g., ``-D AMDGPU_TARGETS=gfx908:xnack-``)
+* Use lots of threads (e.g., ``-j32``)
+* If they aren't needed, specify either ``HIPTENSOR_BUILD_TESTS`` or ``HIPTENSOR_BUILD_SAMPLES`` as OFF to disable client builds.
+* During the ``make`` command, build a specific target, e.g: ``logger_test``.
+
+Test run lengths
+^^^^^^^^^^^^^^^^^
+
+Depending on the resources available to the machine running the selected tests, hipTensor test runtimes can be on the order of an hour or more. Here are some things you can do to reduce run-times:
+
+* CTest will invoke the entire test suite. You may invoke tests individually by name.
+* Use GoogleTest filters, targeting specific test cases:
 
 .. code-block:: bash
 
-    cd docs
+    <test_exe> --gtest_filter=*name_filter*
 
-    sudo apt-get update
-    sudo apt-get install doxygen
-    sudo apt-get install texlive-latex-base texlive-latex-extra
+* Manually adjust the test cases coverage. Using your favorite text editor, you can modify test YAML configs to affect the test parameter coverage.
```

Comment:
Should we specify somewhere that CK only support up to 2G elements in contraction?